homeassistant/components/insteon/climate.py

```diff
@@ -90,7 +95,7 @@ def supported_features(self):
         return SUPPORTED_FEATURES
 
     @property
-    def temperature_unit(self) -> str:
+    def temperature_unit(self) -> UnitTemperatureT:
```

Comment:
```suggestion
    def temperature_unit(self) -> str:
```

Reply:
I am sorry but accepting your suggestion causes:
homeassistant/components/insteon/climate.py:98: error: Return type "str" of "temperature_unit" incompatible with return type "UnitTemperatureT" in supertype "ClimateEntity"  [override]

And I thought UnitTemperatureT is the better type, instead of changing in ClimateEntity, but I can do that if that is your preference.

Reply:
You can you `str`, the issue should resolve itself once #53226 is accepted / merged.

Reply:
That PR has been merged, so this needs to rebase in order to be able to use the suggestion.

Reply:
Rebased and corrected.